django/contrib/staticfiles/management/commands/collectstatic.py

```diff
@@ -21,6 +21,8 @@ class Command(BaseCommand):
     """
     help = "Collect static files in a single location."
     requires_system_checks = False
+    overwrite_warning_msg = u"This will overwrite existing files!\n"
+    delete_warning_msg = u"This will DELETE ALL FILES in this location!\n"
```

Comment:
You can remove these `u` prefixes as they are made redundant by `from __future__ import unicode_literals`